src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -337,6 +342,24 @@ impl LocalStateStore for LocalHummockStorage {
         .await
     }
 
+    async fn try_flush(&mut self) -> StorageResult<()> {
+        if self.mem_table.kv_size.size() > self.mem_table_spill_threshold {
+            tracing::info!(
+                "The size of mem table exceeds {} Mb and spill occurs. table_id {}",
+                self.mem_table_spill_threshold >> 20,
```

Comment:
How about logging the actual memtable size instead of the threshold? I think this is more informative.

Reply:
It's ok, and as we do chunk-level spill, I guess the memtable size will not be much larger than threshold.